packages/xstate-graph/src/graph.ts

```diff
@@ -366,7 +366,7 @@ export function getPathFromEvents<
     defaultMachineStateOptions as any
   );
 
-  const { serializeState } = optionsWithDefaults;
```

Comment:
does it shadow a variable of the same name from the outer scope?

Reply:
Will double check